Apps/W1/ShopifyConnector/app/Base/Page Extensions/ShpfyAccMgrRoleCtrExt.PageExt.al

```diff
@@ -0,0 +1,86 @@
+/// <summary>
```

Comment:
Please remove this RC, we don't support it.

Reply:
Updated with commit "Object Renaming"